NEWS.md

```diff
@@ -3,6 +3,8 @@
 
 Install the development version of reticulate with `remotes::install_github("rstudio/reticulate")`.
 
+- `input()` and `raw_input()` now work as expected within `repl_python()`. (#610)
```

Comment:
Will this also make them behave correctly when running from any piece of Python code? (e.g. code in a .py script or Python package)

Reply:
Indeed, it works in all of these contexts. (we override the `input` builtin so our hook gets called in any context where Python's `input()` might get called)